src/main.js

```diff
@@ -19,16 +24,62 @@ import { listFrameworks as list, hasFramework as has, getFramework as get } from
  * @property {string} directory - Relative path to the directory where files are built
  */
 
+/**
```

Comment:
This syntax works, but just FYI, that you can write them in a *.d.ts file as well. See https://github.com/netlify/plugins/blob/main/types/plugins.d.ts and where it gets referenced here https://github.com/netlify/plugins/blob/main/bin/sync_plugins_to_cms.js#L13